redis/cluster.py

```diff
@@ -167,6 +167,13 @@ def parse_cluster_myshardid(resp, **options):
     "ssl_password",
     "unix_socket_path",
     "username",
+    "cache_enable",
+    "client_cache",
+    "cache_max_size",
```

Comment:
IMHO I think we have enough duplication with all these. Part of another issue to clean up / tech debt?